code/game/machinery/computer/buildandrepair.dm

```diff
@@ -376,6 +376,21 @@
 	to_chat(user, "<span class='warning'>You fiddle with a few random fuses but can't find a routing that doesn't short the board.</span>")
 	return
 
+
+/obj/item/weapon/circuitboard/fishtank/solder_improve(mob/user as mob)
```

Comment:
Please remove `as mob`, it's not useful.

Reply:
removed from all the solder_improve instances